packages/kinvey-js-sdk/src/kmd.js

```diff
@@ -0,0 +1,111 @@
+import isPlainObject from 'lodash/isPlainObject';
+
```

Comment:
Similar question as with `acl`: if we are changing this class form `metadata` to `kmd`, is this a breaking change?